src/analysis/functions.rs

```diff
@@ -104,6 +104,15 @@ impl Info {
         self.new_name.as_ref().unwrap_or(&self.name)
     }
 
+    pub fn is_special(&self) -> bool {
+        self.codegen_name()
+            .trim_end_matches('_')
+            .split('_')
+            .last()
+            .map(|i| i.parse::<special_functions::Type>().is_ok())
+            .unwrap_or(false)
+    }
```

Comment:
This should be `rsplit('_').first() to grab the last item directly. Note that these functions always return the string remainder, so `""` or any other string not containing `_` yields exactly one item (but at least it can easily be parsed as special function).

Reply:
The suggestion doesn't quiet work because rsplit returns a `RSplit` which doesn't have a first method :-) first is unique to slices. I could use get but I don't see what would be the differences here? :) at the end you get the same result

Reply:
Mistyped, `.first()` makes no sense on iterators, that returns the same as `.next()` :wink